internal/healthchecks/healthchecks.go

```diff
@@ -66,20 +68,20 @@ func LogHealthCheckResults(healthCheckResults []HealthCheckResult, infoLogger fu
 	}
 }
 
-func CreateHealthChecksLogger(logDir string) (logger *log.Logger, closer func()) {
+func CreateHealthChecksLogger(logDir string) (logger *logs.FileLogger, closer func()) {
 	path := filepath.Join(logDir, healthChecksLogFile)
 	// Make sure the directory exists before writing the file.
 	if err := os.MkdirAll(logDir, 0755); err != nil {
 		log.Printf("failed to create directory for %q: %v", path, err)
-		return log.Default(), func() {}
+		return logs.Default(), func() {}
 	}
 	file, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
 	if err != nil {
 		log.Printf("failed to open health checks log file %q: %v", path, err)
-		return log.Default(), func() {}
+		return logs.Default(), func() {}
```

Comment:
Should we keep the standard golang logger `Default()` instead of this one (when the `health-check.log` can't be created) ? Do we want structured logs in stdout in this case ?

Reply:
IMO we maintain consistency and print structured logs to stdout. It won't get parsed but it will be the same content as writing to the file.

Reply:
I agree with maintaining consistency of adding structured logs to `stdout` too.